utilities/claim/claim.js

```diff
@@ -160,49 +168,64 @@ async function initiateRound(privateKey, { ethRegistryAddress, ethTokenAddress,
       gas = await ethRewardsManagerContract.methods.transferToSolana(arbiterFee, nonce).estimateGas()
       const tx = await ethRewardsManagerContract.methods.transferToSolana(arbiterFee, nonce).send({
         from: accountAddress,
-        gas
+        gas,
+        gasPrice
       })
 
       const txHash = tx.transactionHash
       console.log('Successfully transferred to wormhole, transaction:', txHash)
+      // Append transaction hash to file
+      fs.appendFileSync(wormholeFile, txHash + '\n')
```

Comment:
I believe both this function and its read counterpart can throw in certain scenarios. Does it make sense for us to catch in both places?

Reply:
probably not here, we should just throw and the job will re-run (it's run on a 5 min cron)